src/components/organisms/data-table/types.ts

```diff
@@ -76,4 +79,10 @@ export type DataTableToolbarProps = {
   };
 };
 
+export type DataTableStatusComponentProps = {
+  filters: TableFilters;
+};
```

Comment:
Table Status component should accept filters allowed.

```suggestion
export type DataTableStatusComponentProps = {
  filters: TableFilters;
  filtersAllowed?: {
      [filterName in keyof TableFilters['get']]?: boolean;
  }
};
```